osu.Game/Overlays/Changelog/ChangelogContent.cs

```diff
@@ -7,20 +7,29 @@
 using osu.Framework.Graphics.Containers;
 using osu.Game.Online.API.Requests.Responses;
 using System;
+using osu.Framework.Graphics.Cursor;
 
 namespace osu.Game.Overlays.Changelog
 {
-    public class ChangelogContent : FillFlowContainer
+    public class ChangelogContent : PopoverContainer
```

Comment:
I'm really not sure about having to add these all over the place. Did we discuss having a global one which can be the fallback case for whenever a local one is not required? Is there an issue with doing that?

The reason I bring this up is that this weird interaction happens due to the local `PopOverContainer` being padding too close:

https://user-images.githubusercontent.com/191335/197678961-255a6418-e1ae-43ac-9316-8b035b89d739.mp4

Reply:
I also had a question "why popover container isn't global". I added game-wide one inside `TooltipContainer`. Nothing seems to have broken from this.

Reply:
Bit hesitant to have that change bundled with this PR. At very least we'll want to go through all other usages of popovers as a follow-up and remove any local implementations that are no longer necessary (ie. anything that isn't in a scroll container and relying on the popover being localised to better track scroll).